scour/scour.py

```diff
@@ -3597,9 +3597,7 @@ def xmlnsUnused(prefix, namespace):
 
 
 # used mostly by unit tests
-# input is a filename
-# returns the minidom doc representation of the SVG
-def scourXmlFile(filename, options=None):
+def scourXmlFileAndReturnString(filename, options=None):
```

Comment:
As a matter of fact we'll probably remove `scourXmlFile()` in favour of some (yet to be determined) public API eventually.

Could you rewrite the tests to use `scourString` directly instead of adding yet another "legacy" function?